xenmgr/Vm/Monitor.hs

```diff
@@ -160,22 +167,33 @@ insertDefaultEvents m = let uuid = vmm_uuid m in do
           _ -> return ()
     whenPowerState _ = return ()
 
+    whenNetState state backend = 
+        case state of
+            30  -> manageFrontVifs False (ND.netbackToUuid backend) --Network backend disconnected, handle it...
+            100 -> manageFrontVifs True (ND.netbackToUuid backend)
```

Comment:
They're integers network manager uses to enumerate device state.  The UI was already hooked into these state updates, so it has them defined as constants in https://github.com/OpenXT/toolstack-data/blob/master/dist/script/constants.js#L205.

I'd be fine with macro-ing them to something descriptive.